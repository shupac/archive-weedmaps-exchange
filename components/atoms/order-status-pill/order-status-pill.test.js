import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import StatusPill from './';

describe('Styled Link', () => {
  it('should render the status pill', () => {
    const component = <StatusPill status="canceled" />;
    const wrapper = shallow(component);
    expect(wrapper.exists()).toEqual(true);
    const pill = renderer.create(component);
    expect(pill).toMatchSnapshot();
  });
});
