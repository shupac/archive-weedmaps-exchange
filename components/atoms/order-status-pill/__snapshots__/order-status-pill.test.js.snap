// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Link should render the status pill 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 24px;
  width: 120px;
  border: 1px solid #D0021B;
  border-radius: 4px;
  color: #D0021B;
  line-height: 0.75rem;
  font-size: 0.75rem;
  font-weight: 600;
  text-transform: uppercase;
}

<div
  className="c0"
>
  cancelled
</div>
`;
